cmake/Modules/EttercapLibCheck.cmake

```diff
@@ -122,7 +122,8 @@ if(ENABLE_PLUGINS)
 #        set(HAVE_PLUGINS 1)
 #        set(EC_LIBS ${EC_LIBS} ${HAVE_LTDL})
 #    endif(HAVE_LTDL)
-    CHECK_LIBRARY_EXISTS(dl dlopen "dlfcn.h" HAVE_DLOPEN)
+    #CHECK_LIBRARY_EXISTS(dl dlopen "dlfcn.h" HAVE_DLOPEN)
+    CHECK_FUNCTION_EXISTS(dlopen HAVE_DLOPEN)
```

Comment:
mmmm I found probably this CMAKE_DL_LIBS that might do all for us
http://www.cmake.org/cmake/help/v3.0/variable/CMAKE_DL_LIBS.html

what do you think about?